manifests/profile/master/postgres.pp

```diff
@@ -0,0 +1,56 @@
+# @summary Aplly this class to an agent running pe-postgresql to collect postgres metrics
+#
+# @param query_interval
+#   How often to run the queries in minutes.  Defaults to 10 minutes.
+#
+# @param postgres_host
+#   The FQDN of the postgres host.  Defaults to the FQDN of the server where the profile is applied
+#
+# @param port
+#   The port that the postgres service listens on.  Defaults to 5432
+define puppet_metrics_dashboard::profile::master::postgres (
+  Variant[String,Tuple[String, Integer]] $postgres_host = $facts['networking']['fqdn'],
```

Comment:
Unneeded since this is being applied to the host running postgres (see notes below). You would only need to define this if hitting a remote postgres server AFAIK.